templates/_layouts/html.twig

```diff
@@ -60,6 +60,23 @@
           document.documentElement.className = document.documentElement.className.replace('has-no-js', 'has-js');
         </script>
         {% block head_end %}{% endblock %}
+
+        {# GA4 Google Analytics tracking #}
+        {#
+          TODO: get a Google Analytics 4 Property ID from the .env settings or authors to enable tracking
```

Comment:
_TODO: Get a Google Analytics 4 Property ID from the control panel to enable tracking._

We’re already getting it from `.env` in line 68. 🤷

Reply:
Updated the comment for both sources to get the ID, _or authors ..._ was a bit unclear.